dev/AppNotifications/AppNotificationManager.cpp

```diff
@@ -80,8 +90,11 @@ namespace winrt::Microsoft::Windows::AppNotifications::implementation
             THROW_IF_FAILED(notificationPlatform->AddToastRegistrationMapping(processName.get(), toastAppId.c_str()));
         }
 
+        winrt::guid registeredClsid{ GUID_NULL };
+        THROW_IF_FAILED(PushNotificationHelpers::GetComRegistrationFromRegistry(expectedAppServerArgs.data(), registeredClsid));
```

Comment:
Why is this code path getting executed for unpackaged apps? It wouldnt work right?

Reply:
The if checks aren't accurate imho.

Reply:
This may be solved if GetComRegistrationFromRegistry returns a GUID instead of HRESULT. I would take this as a bug fix (if we change this, then we need to make changes in PushNotificationManager as well).

Reply:
Don't hide functionality in an API please as discussed. Do the IsPackaged check here. Not in the API

Reply:
@sharath2727 said that he will fix this now. If it is really a big change, then sure we can move it to a bug.